### PR TITLE
Fade In Background Image TFAC-198

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

**Comment by amaliwanag2** [comment]
@kmjennison this is what i did for manual testing:
* couldn't test vercel deployment
* tested locally with javascript disabled and network throttled to slow 3g, looked fine IMO
* tested that the fade works when background image changes after app is already loaded

**Comment by kmjennison** [comment]
There seems to be some compatibility issue. It looks great in general, but on Firefox when [deployed](https://tab-web-g9gqykivj.vercel.app/newtab/), the image is fairly delayed and doesn't fade in. It works fine on FF when JS is disabled. Chrome works great.

**Comment by kmjennison** [comment]
To attempt to fix this, I've tried:
* Looking for unexpected prop changes that would be unmounting -- I don't see any
* Using inline styling for the `backgroundImage` property -- doesn't fix the problem
* Using MUI's `Fade` component -- this relies on JS, so it'll be too slow
* Using `react-transition-group`'s `CSSTransition` -- this relies on JS, so it'll be too slow

My suspicion is this is related to how MUI styles are injected during SSR and then loaded in JS. Possibly the switch from CSS to CSS-in-JS is causing some unexpected rendering in Firefox. **We could try using a CSS animation outside of MUI.**

Alternatively, maybe this is just a performance issue with Firefox.

**Comment by kmjennison** [comment]
I tried a CSS animation using CSS modules, but it had the same issue. Starting to suspect it's just a performance difference between the browsers.

File: src/components/UserBackgroundImage.js [hunk 1]
```diff
@@ -9,7 +9,12 @@ import SetBackgroundDailyImageMutation from 'src/utils/mutations/SetBackgroundDa
 
 dayjs.extend(isToday)
 const useStyles = makeStyles(() => ({
+  '@keyframes fadeIn': {
+    from: { opacity: 0 },
+    to: { opacity: 1 },
+  },
   background: {
+    animation: '$fadeIn 0.5s ease',
```
Comment (kmjennison) [inline src/components/UserBackgroundImage.js:17]: Let's move the animation to the root element so that it fades in the "tint" component, too.

**Review by kmjennison — CHANGES_REQUESTED** [review]
@amaliwanag2 Would love your thoughts on the fade issue I'm encountering in Firefox. (See my barrage of comments/commits—no luck fixing it.) Maybe we just leave it for now.

**Comment by kmjennison** [comment]
I wonder if the performance hit comes during React hydration (e.g., is there a hydration mismatch?). When JS is enabled, I see a full page repaint after mount.

**Review by kmjennison — APPROVED** [review]
UX question for the future: should we fade the whole page (like this PR) or just the image? We should do whatever is perceived smoother/faster.

**Pull request merged**